clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestCloneVeniceZKPaths.java

```diff
@@ -0,0 +1,174 @@
+package com.linkedin.venice;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class TestCloneVeniceZKPaths {
+  private final Set<String> CLUSTERS = new HashSet<>(Arrays.asList("cluster1", "cluster2"));
+  private final String BASE_PATH = "/venice-parent";
+
+  @Test
+  public void mainTest() {
+    testBuildRequiredPathsTree();
+    testGetVenicePaths();
+    testExtractVenicePaths();
+    // testCloneVenicePaths();
+  }
+
+  public void testCloneVenicePaths() {
+    // TODO write tests for cloneVenicePaths()
+  }
+
+  // tests AdminTool.extractVeniceZKPathsFromFile() as well as CloneVeniceZKPaths.extractVenicePaths()
+  public void testExtractVenicePaths() {
+    // change this variable to path of where Venice project is stored
+    String directory = "/Users/tonchen/Desktop/";
+    String inputPath = directory + "venice/clients/venice-admin-tool/src/test/java/com/linkedin/venice/zk_paths.txt";
+    String outputPath =
+        directory + "venice/clients/venice-admin-tool/src/test/java/com/linkedin/venice/venice_paths.txt";
+    String[] args = { "--extract-venice-zk-paths-from-file", "--zk-paths-file", inputPath, "--venice-paths-file",
+        outputPath, "--cluster-list", "cluster1, cluster2", "--base-path", BASE_PATH };
+    try {
+      AdminTool.main(args);
+    } catch (Exception e) {
+      System.out.println(e.getMessage());
+    }
+    // manually check venice_paths.txt and verify that all (21) Venice-specific paths are extracted correctly
+  }
+
+  public void testGetVenicePaths() {
+    ArrayList<String> zkPaths = getPaths();
+    Set<String> requiredPaths = CloneVeniceZKPaths.getRequiredPaths();
+    ArrayList<String> venicePaths = CloneVeniceZKPaths.getVenicePaths(zkPaths, CLUSTERS, BASE_PATH, requiredPaths);
+    testVenicePathsAsserts(venicePaths);
+  }
+
+  public void testVenicePathsAsserts(ArrayList<String> venicePaths) {
+    Assert.assertEquals(venicePaths.size(), 21);
+    Assert.assertFalse(venicePaths.contains("/venice-parent"));
```

Comment:
This is incorrect. We need to have `/venice-parent` too. This is because the Znode contents of that path could be important